src/Web/Grand.Web.Admin/Areas/Admin/Views/Customer/_CreateOrUpdate.TabInfo.cshtml

```diff
@@ -33,6 +33,9 @@
     </script>
 }
     <div class="form-horizontal">
+        @*Add dummy text and password fields to avoid autofill issue on create customer page*@
+         <input type="text" style="opacity: 0;position: absolute;">
+         <input type="password" style="opacity: 0;position: absolute;">
```

Comment:
I don't think so is a good solution

Reply:
@KrzysztofPajak  We tried adding Autocomplete='off' attribute in input elements but that is getting ignored by the Chrome and all major browsers.

Reply:
you can pass type="password" autocomplete="new-password"

at least the password will never be auto populated